examples/notebooks/beam-ml/run_inference_basic.ipynb

```diff
@@ -1362,6 +1308,408 @@
         "      | beam.Map(print)\n",
         "  )\n"
       ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# TensorFlow - Examples\n",
```

Comment:
Nit: Right now we have all 3 frameworks labeled differently (`# PyTorch`, `# Sklearn implementation of RunInference API.`, `# TensorFlow - Examples`). I don't really care which we choose, but can we consolidate on a pattern?

Reply:
Ok. I consolodated the titles on Beam RunInference for Framework